Automattic/wp-calypso
# 3 mins review: LiB pattern picker: use MShots for better perf

#### Proposed Changes

* This uses mshots for previews instead of iframes to make the page faster. Loading iframes is 2-3 MB each. These screenshots are 100-200KB each.

#### Testing Instructions

1. Go to https://container-nifty-mendeleev.calypso.live/setup?flow=link-in-bio
2. Check the pattern picking step. It should be responsive and works as expected.
3. Make sure to test on simulated mobile.
